docs/ha-mode.md

```diff
@@ -21,8 +21,7 @@ The `etcd_access_endpoint` fact provides an access pattern for clients. And the
 `etcd_multiaccess` (defaults to `false`) group var controlls that behavior.
 When enabled, it makes deployed components to access the etcd cluster members
 directly: `http://ip1:2379, http://ip2:2379,...`. This mode assumes the clients
-do a loadbalancing and handle HA for connections. Note, a pod definition of a
-flannel networking plugin always uses a single `--etcd-server` endpoint!
+do a loadbalancing and handle HA for connections.
 
```

Comment:
Wait this still looks out of sync.  etcd_multiaccess defaults to false in the docs, but you can see below that it defaults to true.  This is still out of sync.